### DISCUSSION TITLE
How to change font

### DISCUSSION THREAD

**Discussion by vicrucann** [posquit0/Awesome-CV#495] [Q&A]
I'd like to try some other common font, e.g., Times New Roman. Is there an easy way to change that?

**Comment by sylhare** [comment]
Allo @vicrucann ,

If you want to change the font, I would suggest to use these commandes (they're shown in the awesome-cv.cls and can be tweaked). 
```
\footerfont{Times New Roman}
\bodyfont{Times New Roman}
\bodyfontlight{Times New Roman}
```

However you have to make sure the new font you want to use is in the right directory or use a 
`\RequirePackage[default,opentype]{sourcesanspro}` like the one in the awesome-cv.cls like you would use `\sourcesanspro` after, but I don't know exactly how to use it.

Hope it helps.

**Comment by alishir** [comment]
You can define a custom font family and use it different sections, here is how I use `Montserrat` font for the cover letter body:

```LaTeX
%-------------------------------------------------------------------------------
%                Configuration for fonts
%-------------------------------------------------------------------------------

\newfontfamily\alishirfont[
  Path=\@fontdir,
  UprightFont=*-Regular,
  ItalicFont=*-Italic,
  BoldFont=*-Bold,
  BoldItalicFont=*-BoldItalic,
]{Montserrat}

% For elements of cover letter

\newcommand*{\mylettertextstyle}{\fontsize{8pt}{1.4em}\alishirfont\selectfont\color{graytext}}

```

p.s.: You should copy font files in the `font` directory.

**Reply by sbrunswi** [reply]
where did you find the font files? Do I also adjust the font family? When I do what you do it still keeps the standard font.

**Comment by ghost** [comment]
Hello everybody, I am still interested in resolving this issue.
I am not a proficient coder in TeX, but I am very interested in understanding how to properly change fonts.
Like, I got that you have to add the .tff files to the font directory and adjust the code in the awesome-cv.cls.
But this alone doesn't fix the problem for me.
Is there anyone who understands this and would like to share an easy explanation (or maybe some easy code example)?
Thank you so much!

**Comment by zxdawn** [comment]
@sylhare I got this error when adding three lines mentioned by you:

```
LaTeX Error: Missing \begin{document}. [\footerfont{T]
```

Any idea how to fix it?

**Comment by zxdawn** [comment]
Well, I found the [solution](https://tex.stackexchange.com/a/345806/283088) on StackOverflow:

Add these just before `\begin{document}`
```
\let\headerfont\rmfamily
\let\headerfontlight\rmfamily
\let\footerfont\rmfamily
\let\bodyfont\rmfamily
\let\bodyfontlight\rmfamily
```

**Comment by arudaev** [comment]
How does one change the font size? Or make the font a bit thicker

**Comment by chulao14** [comment]
I'm trying to change the font style but I still can't do it

**Comment by arudaev** [comment]
> I'm trying to change the font style but I still can't do it

I at some point succeeded, but the end result didn't look well. I suggest keeping to the defaults

**Comment by arudaev** [comment]
The font size can be changed inside `awesome-cv.cls` lines **191-234**, which is just a trial and error when finding the correct line to edit.

**Comment by arudaev** [comment]
![image](https://github.com/posquit0/Awesome-CV/assets/66517969/8e86de26-535a-4bd0-9593-334fbc476989)
